### PR TITLE
Add Zuul as CI

### PR DESCRIPTION
**Pull request by lego963**
Add `.zuul.yaml` config file

Add `.editorconfig`

Update `dns_nameservers` var

Update `README.md`

### CODE REVIEW

**Comment by lego963** [comment]
recheck

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+insert_final_newline = true
+max_line_length = 80
+trim_trailing_whitespace = true
+
+[*.{tf,tfvars}]
+indent_size = 2
+indent_style = space
+
+[*.md]
+max_line_length = 0
```
Comment (outcatcher) [inline .editorconfig]: `max_line_length = 0` ― is this a valid value?

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+max_line_length = 80
```
Comment (outcatcher) [inline .editorconfig]: Maybe use `100` for `max_line_length`?

File: README.md [hunk 3]
```diff
@@ -62,31 +62,31 @@ No modules.
...
+| <a name="input_vpc_cidr"></a> [vpc\_cidr](#input\_vpc\_cidr)                            | IP range for the VPC                                       | `string`       | `null`                                                       |   yes    |
+| <a name="input_subnet_cidr"></a> [subnet\_cidr](#input\_subnet\_cidr)                   | IP range for the subnet                                    | `string`       | `null`                                                       |   yes    |
+| <a name="input_gateway_ip"></a> [gateway\_ip](#input\_gateway\_ip)                      | The gateway of the subnet                                  | `string`       | `null`                                                       |   yes    |
+| <a name="input_dns_nameservers"></a> [dns\_nameservers](#input\_dns\_nameservers)       | An array of DNS name servers used by hosts in the subnet   | `list(string)` | <pre>[<br>  "100.125.4.25",<br>  "100.125.129.199"<br>]<pre> |    no    |
```
Comment (outcatcher) [inline README.md]: Maybe we should use `8.8.8.8` or `1.1.1.1` as an additional name server in case there are problems with internal ones.

**Review by outcatcher — CHANGES_REQUESTED** [review]

File: .editorconfig (hunk 1 above)
Comment (lego963) [reply .editorconfig]: removed

File: .editorconfig (hunk 2 above)
Comment (lego963) [reply .editorconfig]: done

File: README.md (hunk 3 above)
Comment (lego963) [reply README.md]: add 8.8.8.8

**Review by outcatcher — DISMISSED** [review]

**Review by outcatcher — APPROVED** [review]

**Comment by lego963** [comment]
regate

**Pull request merged**